src/shared/search.ts

```diff
@@ -11,11 +11,12 @@ export interface SearchResult {
   rootDirectory: string
 }
 
-function getDefaultGlobOptions(): glob.GlobOptions {
+function getDefaultGlobOptions(includeHiddenFiles: boolean): glob.GlobOptions {
```

Comment:
nit: Can we just call this excludeHiddenFiles like we do in the glob options?

Reply:
Going to leave this as-is since we've back-ported the changes with this variable name. We can fix it in a follow-up if it gets confusing.